src/Datadog.Trace.ClrProfiler.Native/calltarget_tokens.cpp

```diff
@@ -136,7 +136,7 @@ HRESULT CallTargetTokens::EnsureBaseCalltargetTokens()
     // *** Ensure profiler assembly ref
     if (profilerAssemblyRef == mdAssemblyRefNil)
     {
-        const AssemblyReference assemblyReference = managed_profiler_full_assembly_version;
+        const AssemblyReference assemblyReference = trace::AssemblyReference(managed_profiler_full_assembly_version);
```

Comment:
Should this also get the cached AssemblyReference, like this?

```suggestion
        const AssemblyReference assemblyReference = trace::AssemblyReference::GetFromCache(managed_profiler_full_assembly_version);
```

Reply:
Oh!, you're right, I missed the `GetFromCache` part 🤦🏻